src/pages/about/governance.md

```diff
@@ -1,44 +1,51 @@
----
-id: governance
----
-
 # Tauri Governance
 
-## Sustainability
+One of the main goals of the organizational structure of Tauri is to guarantee the sustainability, health and well-being of its contributors. [The Tauri Programme within the Commons Conservancy](https://dracc.commonsconservancy.org/0035/) was established to facilitate an open, transparent and efficient governance process throughout the future development of the Tauri and its auxiliary materials.
```

Comment:
@Beanow I tweaked the grammer on that suggestion.

I think I'd use any of the "why's" in a blog post about these changes instead of here (and reference that blog post on this page). To Perry's point we should think of what's the point of this page, it's to serve as a quick tl;dr reference, the blog post can be a deep-dive into the _why_.